ssca-manager/chart/templates/config.yaml

```
@@ -16,4 +16,12 @@ data:
   MONGO_INDEX_MANAGER_MODE: AUTO
   NG_MANAGER_BASE_URL: http://ng-manager.{{ .Release.Namespace }}.svc.cluster.local:7090
   DEPLOY_MODE: KUBERNETES_ONPREM
-  STACK_DRIVER_LOGGING_ENABLED: '{{ .Values.global.stackDriverLoggingEnabled }}'
\ No newline at end of file
+  STACK_DRIVER_LOGGING_ENABLED: '{{ .Values.global.stackDriverLoggingEnabled }}'
+  S3_BUCKET: {{ .Values.s3.sbomBucketName }}
+  {{ - if .Values.s3.endpoint }}
+  S3_ENDPOINT: { { .Values.s3.endpoint | default .Values.s3.defaultMinioUrl } }
```

This default value will never be used since this is inside an if statement.
```suggestion
  S3_ENDPOINT: {{ .Values.s3.endpoint  }}
```